### DISCUSSION TITLE
2024-05-25 (토)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#38] [데일리스크럼]
아래 양식 복사해서 댓글로 작성해주시면 됩니다😃
```
## 이름
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
## 이서인
### 완료한일
- ImageBox 컴포넌트 구현
- BottomModal 컴포넌트 구현

### 해야할 일

### 어려운 일

**Comment by q45402sk** [comment]
## 기송은
### 완료한일

### 해야할 일
- card 컴포넌트 구현 
- 찜 컴포넌트 구현 

### 어려운 점 
- 낙관적 업데이트 부분이 어려워서 카드 컴포넌트 먼저 구현하고 있습니다.

**Reply by Seoin02** [reply]
낙관적 업데이트 어려울 것 같아요ㅠㅠ💪🏻💪🏻

**Comment by devwqc** [comment]
## 정봉찬
### 완료한일
- AWS CI/CD 무중단 배포
- GitHub Actions + Vercel을 활용한 PR 미리보기

### 해야할 일
- 잠시 보류된 Nav 작업 진행

### 어려운 일
- 배포가 되게 중요하다고 생각해서 먼저 배포 준비하느라 공통 컴포넌트 작업이 되지 않았습니다. 죄송합니다.

**Comment by minyoung0503** [comment]
## 김민영
### 완료한일

### 해야할 일
* 리뷰 박스 컴포넌트 구현

### 어려운 일
